GitUI/AutoCompletion/CommitAutoCompleteProvider.cs

```diff
@@ -137,8 +139,7 @@ private static IEnumerable<string> ReadOrInitializeAutoCompleteRegexes()
             return regexes;
         }
 
-        [CanBeNull]
-        private static async Task<string> GetChangedFileTextAsync(GitModule module, GitItemStatus file)
+        private static async Task<string?> GetChangedFileTextAsync(GitModule module, GitItemStatus file)
```

Comment:
This annotation was previously incorrect. It should have been `ItemCanBeNull`. With NRT the compiler will enforce correctness of annotations.